src/fmelt.c

```diff
@@ -91,19 +91,25 @@ SEXP concat(SEXP vec, SEXP idx) {
 
 // deal with measure.vars of type VECSXP
 SEXP measurelist(SEXP measure, SEXP dtnames) {
-  int i, n=length(measure), protecti=0;
```

Comment:
I don't really see any work done in fmelt since 1.12.8... I guess that means something upstream changed? Maybe improvements to `rchk` / R-devel changes?

Reply:
Yes, most likely `rchk` changes I'd guess. I am using latest rchk fetched yesterday from master:  https://github.com/kalibera/rchk/commits/master.

Reply:
Ah, definitely a version change on `llvm` sounds like a likely candidate, thanks

Reply:
Actaully, fmelt.c did come up last time and I skipped it with comments here:  https://github.com/Rdatatable/data.table/pull/3867#issuecomment-531382945.  Looks like `rchk` is getting better at not showing us the false positives, and I'm getting better at guessing how rchk likes things too.